server/serverAPI/routes/RecipeRoute.ts

```diff
@@ -21,5 +20,5 @@ const recipeController = new RecipeController(
     )
 );
 
-recipeRoute.get('/', recipeController.searchRecipe)
-recipeRoute.get('/recipe/:recipeID',recipeController.getRecipe);
+recipeRoute.get('/', recipeController.getAll)
```

Comment:
Sorry if it's my lack of express understanding, but how can this and foodRoute both specify an endpoint for `/`?

Reply:
ez, then don't. Routes file defines routes in terms of their relative position to the "main route" in express. In App.ts you have main routes for each specific route. For recipe it's recipes for Food it's foods. In their respective files, they are all starting from "/" because that's the starting point relative to their main route. That's how it's supposed to be coded.

Reply:
Ah I see thanks